### DISCUSSION TITLE
2007/07/28/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%837

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#173] [Announcements]
# 2007/07/28/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-7-%E5%A4%96%E6%98%9F%E4%BA%BA%E5%85%A5%E4%BE%B5

我正在路上闲逛的时候，突然手机响了，是正在美国出差的小李打来的。他说他正在看电视呢，新闻里说：最近几天的全球大气活动突然一反常态，接下来几天恐怕会有全球性的大灾害，灾害程度难以预计，只怕一些基本生活供给都要中断了。小李提醒我说，美国这边已经乱了，大家到处再收集粮食呢。这种新闻在中国一定不会播出的，要我赶紧去超市大采购，把未来几天的食物准备好。

https://blog.qizhen.xyz/2007/07/28/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%837

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2007年07月30日 09:35 编辑
    绝大多数的梦都是记不住的，很多时候甚至意识不到自己做过梦。只有这种时候才可能记住自己做的梦：在做梦的当时突然醒过来了，然后立刻把刚才做的梦从头到位回想一遍。    我有个想法，就是做梦不只是潜意识活动，它应该还有个功能类似于计算机中的内存碎片整理、页面调度、硬盘整理修复等的功能：做梦的时候，人脑会检索其高速缓存（负责短时记忆）中的内容，把其中大多是不重要的信息都抛弃掉，重要的记忆被挪到长期记忆的存储设备中去，并且也会检索一部分长期记忆内容，遗忘一部分内容，以腾出空间给新的记忆。    由于人脑会遍历短时记忆的内容，所以就会出现日有所思，也有所梦的现象。    但是它判断一个记忆是否重要的时候，采用的逻辑与清醒状态下分析事情的逻辑是不同的。它还要调出以前的一些记忆用于比较判断其重要性，而且对记忆的组织方式也和白天不无安全相同。所以梦中的好多事情是以象征的形式出现的。有的时候生活中完全不认识的两个人，比如你的同事和同学，会同时出现在一个梦里。有时候，一个人做过的事，会被你在梦里安到另一个人身上。    人脑在把短时记忆转换为长时记忆的过程中，有些记忆会被保存下来，但是它的存储路径却会丢失掉。所以有的时候一件事你怎么想也想不起来，你以为已经把这件事忘掉了。但是在某天做梦的时候，你的大脑整理硬盘的时候，又读到了这个记忆，并且修复了他的查找路径，这样，你会在将来的某一天突然又想起了这件事。

回复
Ansley
2007年07月29日 13:27 编辑
 我做梦怎么就很少记得住呢？以前只记得梦中写了一首很好的诗，梦中告知自己是梦，醒来一定要记着，结果醒来还是忘了。还有，你的梦以后可以作为弗洛伊德2世做研究的原材料。顺便贴一段弗对梦的研究（不知你梦中的密码是什么：－）：
 
夢是一種潛意識的活動，由於人的自我防衛機制把人的本我願望壓抑下去，在潛意識的活動中的主要內容被壓抑的願望並非是直接表達於夢中，而是通過扭曲變作為象徵的形式出現，故夢都是象徵的。佛洛依德認為夢是由「顯夢」（manifest dream-content）及「隱夢」（latent dream-thought）組成的。前者乃夢的表面形式，像經過扭曲與偽裝的「密碼」，以表現隱夢。佛洛依德認為夢可以使睡者的內心渴望滿足；另一個是睡者的求生機制，藉著在夢中重新經歷生命的創痛來保護自己。

回复
yoyo
2007年07月29日 12:54 编辑
整个一科幻电影阿 

回复
花
2007年07月29日 11:15 编辑
哇塞，想象力可真丰富哟 

回复

**Comment by ruanqizhen** [comment]
这本来是我做过的一个梦。我做过不少情节跌宕起伏的梦，但有机会能记录下来的少之又少。于是，我为这个梦添上了细节，改编成了一篇小说。
